BadHabbit/BadHabbit/UserStories/SingnInAppleScreen/SignInAppleViewController.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  SignInAppleViewController.swift
+//  BadHabbit
+//
+//  Created by Halina Kurylchykava on 14.07.23.
+//
+
+import UIKit
+import SnapKit
+import AuthenticationServices
+
+class SignInAppleViewController: UIViewController {
+    private var authorizationButton: ASAuthorizationAppleIDButton!
+    private var imageLogo: UIImageView!
+    private var imageMountains: UIImageView!
+    private var titleLabel: UILabel!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupUI()
+    }
+}
+
+extension SignInAppleViewController {
+
+    private func setupUI() {
+        view.backgroundColor = UIColor(red: 0.81, green: 0.81, blue: 0.81, alpha: 1.00)
+        imageMountains = createImageView(name: "mountains")
+        imageLogo = createImageView(name: "logo")
+        createAuthorizationButton()
+        createTitleLabel()
+        setupConstraints()
+    }
+
+    private func createImageView(name: String) -> UIImageView {
+        let imageName = name
+        let image = UIImage(named: imageName)
+        let imageView = UIImageView(image: image)
+        view.addSubview(imageView)
+        return imageView
+    }
+
+    private func createTitleLabel() {
+        titleLabel = UILabel()
+        view.addSubview(titleLabel)
+        titleLabel.textColor = .black
+        titleLabel.font = UIFont(name: "SFProDisplay-Bold", size: 24)
+        let paragraphStyle = NSMutableParagraphStyle()
+        paragraphStyle.lineHeightMultiple = 1.26
+        let quote = "habit hero"
+        let attributes: [NSAttributedString.Key: Any]  = [
+            .kern: -0.41,
```

Comment:
What is kern?

Reply:
this settings UILabel from figma,
we need to install exactly the same or we can simplify?

Reply:
Properties like this could be skipped. Firstly I'm not sure if all fonts support this feature but it's not necessary to copy everything that sigma defines. Something like paragraphStyle and kern could be skipped

Reply:
changed